libs/nim/nim-groth16-verifier/verify.nim

```diff
@@ -0,0 +1,66 @@
+import
+  bncurve/groups,std/[json,strformat]
```

Comment:
Here and elsewhere, use a single space after commas

Reply:
Fixed